test/test_parse.py

```diff
@@ -539,3 +539,13 @@ def test_func_call(self):
         stmt = parse(query).pop()
         self.assertIsInstance(stmt, nodes.SelectStmt)
         self.assertEqual(stmt.tables(), {'table_one'})
+
+    def test_fingerprint(self):
+        self.assertEqual(
+            fingerprint("SELECT 1"),
```

Comment:
I'd rather test that two equivalent queries have the same fingerprint

Reply:
Your call. Either is fine with me.

Reply:
I also updated #39 with your requests.

Reply:
also updated per your request

Reply:
also added normalization

Reply:
I rather remove this check and add a negative case (some assert not equal).